hummingbird/ml/_parse.py

```diff
@@ -27,6 +27,11 @@
 from .operator_converters import constants
 from .supported import get_sklearn_api_operator_name, get_onnxml_api_operator_name, get_sparkml_api_operator_name
 
+try:
+    from sklearn.ensemble import StackingClassifier, StackingRegressor
+except ImportError:
+    StackingClassifier = None
```

Comment:
Can you please put some comment like below on "StackingClassifier not available in scikit-learn < 0.22" so we dont' forget later when we hopefully clean this

Reply:
yep.